.github/workflows/pre_release.yml

```diff
@@ -49,3 +49,8 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Deprecate Solar Theme on npm
+        run: npm deprecate @refinitiv-ui/solar-theme "Solar theme no longer supported. Use @refinitiv-ui/halo-theme."
```

Comment:
The message should be "Solar theme is deprecated. Consider migrating to @refinitiv-ui/halo-theme instead."